src/Aspire.Hosting/Extensions/ProjectResourceBuilderExtensions.cs

```diff
@@ -100,13 +100,13 @@ public static IResourceBuilder<ProjectResource> WithLaunchProfile(this IResource
             throw new DistributedApplicationException(Resources.LaunchProfileIsSpecifiedButLaunchSettingsFileIsNotPresentExceptionMessage);
         }
 
-        if (!launchSettings.Profiles.TryGetValue(launchProfileName, out var launchProfile))
+        if (!launchSettings.Profiles.TryGetValue(launchProfileName, out var _))
```

Comment:
That syntax defines a variable called `_`. This one discards the value:

```suggestion
        if (!launchSettings.Profiles.TryGetValue(launchProfileName, out _))
```